nautobot_netbox_importer/diffsync/models/abstract.py

```diff
@@ -63,6 +63,10 @@ class DjangoBaseModel(DiffSyncModel):
 
     pk: Union[uuid.UUID, int]
 
+    # known fields in NetBox that should be ignored by the importer.
+    # this is only for logging information, is not critical for the importer.
+    _ignored_fields = ({"id", "choices"})
```

Comment:
`id` I understand - what model has `choices` that we should be ignoring?

Reply:
I was thinking that choices came from enums, but I guess I should look into that more.

Reply:
Sounds right - so maybe add it specifically on the CustomField model?